util/channel.go

```diff
@@ -317,6 +317,29 @@ type ChannelPage struct {
 	Videos []string `yaml:",omitempty"`
 }
 
+// GetChannelPage returns the ChannelPage for the specifid channel
+func (c *Channel) GetChannelPage(projectRoot string) *ChannelPage {
```

Comment:
@jaymickey @j16r is the best practice in go to use full variable names?

```suggestion
func (channel *Channel) GetChannelPage(projectRoot string) *ChannelPage {
```

Reply:
In Go, for methods on a struct the underlying struct variable is usually short.  See an example here: https://github.com/gohugoio/hugo/blob/71b8d8b6a4127acacda8ada08cd61d7bfb18e962/navigation/menu.go#L74